extensions/web3-profile/src/search-web3-profile.tsx

```diff
@@ -25,7 +25,13 @@ export default function Command() {
   return (
     <List isLoading={isLoading} searchBarPlaceholder="ENS search" onSearchTextChange={setSearchTerm} throttle>
       <List.EmptyView
-        title={searchTerm.length ? "No Results" : "Search For ENS Name"}
+        title={
+          isLoading
+            ? "Searching..."
+            : searchTerm.length < 4 && ensSuggestions.length > 0
+            ? "Search For ENS Name"
+            : "No Results"
+        }
```

Comment:
just testing this change, but I'm unable to ever see the `"Search For ENS Name"` case in the UI

was your intent to display that text when the search bar was empty?

Reply:
Ah. could you try again? Also updated the logo, there was a lot of space due to the size - feel free to modify :)

Reply:
The empty string on line 38 is causing the UI (the logo) to shift between states, video below:

https://user-images.githubusercontent.com/372831/188591908-9a480391-3d7d-4100-aa8e-fb2b2df50fe8.mp4

I've made some changes to remove the `description` and update the `title` with necessary instructions. I also refactored the `title` condition to make it easier to reason about. demo below:

https://user-images.githubusercontent.com/372831/188594329-7096bb8b-44bd-427f-8601-f70b4bf2732d.mp4

Reply:
I like it, awesome @peduarte 🔥